src/rpcmining.cpp

```diff
@@ -66,7 +64,7 @@ Value getmininginfo(const Array& params, bool fHelp)
         weight.push_back(Pair("valuesum",   MinerStatus.ValueSum));
         weight.push_back(Pair("legacy",   nWeight/(double)COIN));
         obj.push_back(Pair("stakeweight", weight));
-        obj.push_back(Pair("netstakeweight", nNetworkWeight));
+        obj.push_back(Pair("netstakeweight", GetPoSKernelPS()));
```

Comment:
and here

Reply:
if you feel we its better to use the variable then i can put that back. no worries.